extensions-core/lookups-cached-global/src/main/java/io/druid/server/lookup/namespace/URIExtractionNamespaceCacheFactory.java

```diff
@@ -144,7 +144,7 @@ public String call() throws Exception
                           fmt.print(lastModified),
                           fmt.print(lastCached)
                       );
-                      return version;
+                      return lastVersion;
```

Comment:
My intention was, that in this branch cache is not populated. But if the `version` goes backwards, `preVersion.equals(newVersion)` (https://github.com/druid-io/druid/pull/3609/files/05da84c804bb83905b350145fa26cc57afdb163e#diff-52b91ace82cc677af64b49ff0b736baaR364) will fail, and we will swap existing cache with a clear cache.

If we want to account backward going versions as "newer" versions anyway, this condition: https://github.com/druid-io/druid/pull/3609/files/05da84c804bb83905b350145fa26cc57afdb163e#diff-0410e1aa25b340a16206ef64fcfd2152R138 should be changed.